thirtyinch-lint/build.gradle

```diff
@@ -12,7 +12,7 @@ configurations {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
this can be omitted.
The kotlin plugin take care of it that it will use the same version as the plugin...